providers/infrastructure-vsphere/v1.4.1/cconly/base.yaml

```diff
@@ -454,6 +454,12 @@ spec:
       openAPIV3Schema:
         type: object
         properties: {}
+  - name: controlPlaneTaint
```

Comment:
Because we are changing the variable defintions we should probably bump the version number in the annotation on line 174 `identifier.cc.vmware.com: tanzu-vsphere-default-0.0.1`. Then we should make sure that Shimon on the UX team is aware of this change. Their UI uses this version number to help the UI keep track if it is compatible with this clusterclass.

Reply:
Updated, thanks.